### DISCUSSION TITLE
Spacy list of recognized entities

### DISCUSSION THREAD

**Discussion by IavTavares** [explosion/spaCy#10648] [Help: Other Questions] [answered]
I've scoured Spacy documentation, but I haven't been able to find a complete list of the entities that each of Spacy's NER models are capable of finding...

Any help would be appreciated.

**Comment by thomashacker** [comment, marked as answer]
Hello,
The NER label scheme varies by language and depends heavily on what kind of training data was available (https://github.com/explosion/spaCy/discussions/9147). You can check the labels for the pre-trained models on the [model page](https://spacy.io/models/en#en_core_web_md) (e.g for the `en_core_web_md`)

**Comment by IavTavares** [comment]
Thanks Edward! 😉
